src/panels/lovelace/cards/hui-shopping-list-card.ts

```diff
@@ -0,0 +1,212 @@
+import { html, LitElement } from "@polymer/lit-element";
+import "@polymer/paper-checkbox/paper-checkbox.js";
+import "@polymer/paper-icon-button/paper-icon-button.js";
+import "@polymer/paper-input/paper-input.js";
+import "@polymer/paper-item/paper-icon-item.js";
+import "@polymer/paper-item/paper-item-body.js";
+
+import "../../../components/ha-card.js";
+
+import { HassLocalizeLitMixin } from "../../../mixins/lit-localize-mixin";
+import { HomeAssistant } from "../../../types.js";
+import { LovelaceCard, LovelaceConfig } from "../types.js";
+import { clearItems, fetchItems, addItem, completeItem, saveEdit } from "../../../data/shopping_list";
+
+interface Config extends LovelaceConfig {
+  title?: string;
+}
+
+interface ListItem {
+  id: number;
+  name: string;
+  complete: boolean;
+}
+
+class HuiShoppingListCard extends HassLocalizeLitMixin(LitElement)
+  implements LovelaceCard {
+  protected hass?: HomeAssistant;
+  protected config?: Config;
+  private items?: ListItem[];
+
+  static get properties() {
+    return {
+      hass: {},
+      config: {},
+      items: [],
```

Comment:
items doesn't need to be in here, nor is `[]` a valid options object for lit.

Reply:
options are generally only used when you need to serialize/deserialize things from attributes.